libs/application/templates/healthcare-work-permit/src/fields/CustomFormConclusionSectionField/index.tsx

```diff
@@ -0,0 +1,148 @@
+import {
+  FieldBaseProps,
+  FieldComponents,
+  FieldTypes,
+} from '@island.is/application/types'
+import { Box, Button, Text, PdfViewer } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { FC, useState } from 'react'
+import {
+  AlertMessageFormField,
+  MessageWithLinkButtonFormField,
+  PdfLinkButtonFormField,
+} from '@island.is/application/ui-fields'
+import { coreMessages, formatText } from '@island.is/application/core'
+import * as styles from './styles.css'
+import { confirmation } from '../../lib/messages'
+import { conclusionMessages } from '@island.is/application/ui-forms'
+import { PermitProgram } from '../../lib'
+import { HealthcareWorkPermitAnswers } from '../..'
+
+export const CustomFormConclusionSectionField: FC<
+  React.PropsWithChildren<FieldBaseProps>
+> = (props) => {
+  const { application } = props
```

Comment:
```suggestion
export const CustomFormConclusionSectionField = ({application}: FieldBaseProps) => {
```
Smá nitpic en þetta er læsilegri ritháttur og minni kóði